buildbot/buildbot/worker.py

```diff
@@ -492,15 +492,20 @@ def stage_fpga_execute(db, config):
             # binary or the emulation executable.
             if task['mode'] == 'hw':
                 exe_cmd = ['sudo', 'sh', '-c',
-                           'source /opt/xilinx/xrt/setup.sh ; ./host']
+                           'source /opt/xilinx/xrt/setup.sh ;\
+                            ./{}'.format(config['EXECUTABLE_NAME'])]
             else:
-                exe_cmd = ['sh', '-c', 'cur=`pwd`; cd $AWS_FPGA_REPO_DIR ;\
+                exe_cmd = ['sh','-c','cur=`pwd`; cd $AWS_FPGA_REPO_DIR ;\
```

Comment:
Were these spaces deleted by mistake? :smiley:

Reply:
sorry I didn't notice this! Just fixed it! 😭